config/virtualdesktop/config.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package virtualdesktop
+
+import (
+	"github.com/upbound/upjet/pkg/config"
+)
+
```

Comment:
There is a contradiction in the group names here, under the examples the name `desktopvirtualization` is mentioned, here `virtualdesktop`.

As a group name `virtualdesktop` seems more logical. For this, we can define `const group = "virtualdesktop"` and add `r.ShortGroup` configuration for all resources in the group.

```suggestion

const group = "virtualdesktop"

```
As an example: https://github.com/upbound/provider-azuread/blob/8a3e0bfa3bd229dffddbd14be526a119f121dd1b/config/serviceprincipals/config.go#L5C1-L6C1